### DISCUSSION TITLE
Which version is back-compatible with Vue 2/Nuxt 2.15?

### DISCUSSION THREAD

**Discussion by emilyrbowe** [nolimits4web/swiper#4759] [Q&A] [answered]
Hi, I'd love to use Swiper for a mobile-first web app, but I saw in the documentation that the current version is only compatible with Vue 3. I'm using Nuxt, which as of right now is still running on Vue 2, so I'm hoping a previous release of Swiper is compatible and might work. Thanks in advance for any help!

**Comment by nolimits4web** [comment, marked as answer]
There is no official swiper-vue versions prior to v6. You can use v6 using core JS API https://swiperjs.com/get-started

**Reply by emilyrbowe** [reply]
Thanks! Appreciate you taking the time to answer.

**Comment by viT-1** [comment]
With composition API last version is unofficial can be [supported](https://codepen.io/viT-1/pen/KKELxZj).
